Zend/zend_vm_def.h

```diff
@@ -4675,7 +4675,7 @@ ZEND_VM_HANDLER(63, ZEND_RECV, NUM, ANY)
 		}
 	}
 
-	ZEND_VM_NEXT_OPCODE();
+	ZEND_VM_NEXT_OPCODE_CHECK_EXCEPTION();
```

Comment:
This should be handled the same way as recv_init, i.e. by checking eg(exception) in the verify_arg_type branch.